src/frontend/src/pages/AdminToolsPage/ProjectsConfig/LinkTypeFormModal.tsx

```diff
@@ -12,17 +12,26 @@ import HelpIcon from '@mui/icons-material/Help';
 import { useTheme } from '@mui/material/styles';
 
 interface LinkTypeFormModalProps {
-  showModal: boolean;
+  open: boolean;
   handleClose: () => void;
   defaultValues?: LinkType;
   onSubmit: (data: LinkTypeCreatePayload) => void;
   linkTypes: LinkType[];
+  creatingNew: boolean;
```

Comment:
you don't need this prop, if defaultValues is undefined, you can assume it's a create modal, and if it's defined, it's edit